src/sql/workbench/browser/modelComponents/button.component.ts

```diff
@@ -132,18 +159,34 @@ export default class ButtonComponent extends ComponentWithIconBase<azdata.Button
 
 	protected updateIcon() {
 		if (this.iconPath) {
-			if (!this._iconClass) {
-				super.updateIcon();
-				this._button.icon = this._iconClass + ' icon';
-				// Styling for icon button
-				this._register(attachButtonStyler(this._button, this.themeService, {
-					buttonBackground: Color.transparent.toString(),
-					buttonHoverBackground: Color.transparent.toString(),
-					buttonFocusOutline: focusBorder,
-					buttonForeground: foreground
-				}));
+			if (this.buttonType !== 'Informational') {
```

Comment:
I don't see a reason to duplicate this code, the old one should just work for the new button type.

Reply:
Removed. Turns out I don't need to modified this copied block for info button.